tests/e2e/driver/CheReporter.ts

```diff
@@ -120,6 +121,7 @@ class CheReporter extends mocha.reporters.Spec {
     });
 
     runner.on('fail', async function (test: mocha.Test) {
+      Logger.error(`CheReporter runner.on.fail: ${test.fullTitle()} failed after ${test.duration}ms`);
```

Comment:
Do we really need this information? I mean that as far as I remember it is already in the logs.

Reply:
The failure report at the end does say that the test had failed, but we've had issues with certain parts of the tests failing inconsistently and this could help us debug. I can remove it if you want.